From 69b03a779045f20bdc8edae1cbc57ebf2a075b72 Mon Sep 17 00:00:00 2001
From: Johannes Lahti <johannes.lahti@jollamobile.com>
Date: Fri, 22 Mar 2019 17:20:59 +0200
Subject: [PATCH 1/2] Our mktemp does not support --dry-run option

---
 pkg-diff.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pkg-diff.sh b/pkg-diff.sh
index 72926c2..5a61a3e 100755
--- a/pkg-diff.sh
+++ b/pkg-diff.sh
@@ -285,8 +285,8 @@ diff_two_files()
 
   offset=`sed 's@^.*differ: byte @@;s@,.*@@' < $dfile`
   echo "$file differs at offset '$offset' ($ftype)"
-  po=`mktemp --dry-run $TMPDIR/old.XXX`
-  pn=`mktemp --dry-run $TMPDIR/new.XXX`
+  po=`mktemp -u $TMPDIR/old.XXX`
+  pn=`mktemp -u $TMPDIR/new.XXX`
   mkfifo -m 0600 $po
   mkfifo -m 0600 $pn
   offset=$(( ($offset >> 6) << 6 ))
-- 
2.26.2

